SIL.Windows.Forms.Scripture/VerseControl.cs

```diff
@@ -57,6 +57,8 @@ public partial class VerseControl : UserControl
 		public event EventHandler InvalidBookEntered;
 		/// <summary>Fired when any part of the reference is invalid</summary>
 		public event EventHandler InvalidReferenceEntered;
+		/// <summary>Fired when any part of the pasted reference is invalid</summary>
+		public event EventHandler InvalidReferencePasted;
```

Comment:
Since this is a new public event, it should have an entry in the ### Added section of CHANGELOG.md (and it should have +semver:minor in the comment)